### PR TITLE
Symfony2/3 Valet driver

### PR DESCRIPTION
**Pull request by igorpan**
This driver should work for both Symfony V2 and V3.

It could be split in two separate drivers if necessary

### CODE REVIEW

**Comment by taylorotwell** [comment]
Nice thanks for give it a try later tonigiht.

**Comment by CHH** [comment]
Would be nice to make the front controller somehow configurable. Sometimes one needs to run Symfony apps using the prod front controller.

File: drivers/SymfonyValetDriver.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return bool
+     */
+    public function serves($sitePath, $siteName, $uri)
+    {
+        return file_exists($sitePath.'/web/app_dev.php') &&
+               (file_exists($sitePath.'/bin/symfony_requirements') ||
+                file_exists($sitePath.'/app/SymfonyRequirements.php'));
```
Comment (WouterJ) [inline drivers/SymfonyValetDriver.php]: -1 on these 2 checks. These are just default conventions, nothing forbits you to change the Standard edition directory structure.

Comment (igorpan) [inline drivers/SymfonyValetDriver.php]: But nothing forbids you from changing anything in Symfony, really. You don't need to have a web directory at all. If you go away from it's defaults, you would have to create your own driver either way.

Comment (igorpan) [inline drivers/SymfonyValetDriver.php]: Maybe we could rename this to "SymfonyStandardEditionValetDriver" to be more specific. I don't think it makes sense making anything more generic than this.

**Comment by taylorotwell** [comment]
Tweaks checks a bit and made it so that if "app_dev" doesn't exist it will serve from "app", so that you can at least run through the production front controller if you need to by renaming "app_dev" to "app_dev_bkp" or something. Thanks!

File: drivers/SymfonyValetDriver.php (hunk 1 above)
Comment (ruudk) [inline drivers/SymfonyValetDriver.php]: Just installed Valet on a Symfony2 project without the 2 files. Didn't work. Took me a while to find that this was causing it.

**Pull request merged**